.github/workflows/sundials4py-wheels.yml

```
@@ -0,0 +1,120 @@
+name: sundials4py wheels
+
+permissions:
+  contents: read
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+  merge_group:
+  workflow_dispatch:
+  release:
+    types:
+      - published
+
+jobs:
+  build_sdist:
+    name: Build SDist
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v5
+      with:
+        submodules: true
+
+    - name: Build SDist
+      run: pipx run build --sdist
+
+    - name: Check metadata
+      run: pipx run twine check dist/*
+
+    - uses: actions/upload-artifact@v5
+      with:
+        name: dist-sdist
+        path: dist/*.tar.gz
+
+
+  build_wheels:
+    name: Wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, macos-15-intel, windows-latest]
+
+    steps:
+    - name: Set macOS deployment target
+      if: contains(matrix.os, 'macos')
+      run: echo "MACOSX_DEPLOYMENT_TARGET=$(sw_vers -productVersion | cut -d '.' -f 1-2)" >> $GITHUB_ENV
+
+    - uses: actions/checkout@v5
+      with:
+        submodules: true
+
+    - uses: pypa/cibuildwheel@v2.22
+      env:
+        CIBW_TEST_COMMAND: "pytest {project}/bindings/sundials4py/test"
+
+    - name: Upload wheels
+      uses: actions/upload-artifact@v5
+      with:
+        path: wheelhouse/*.whl
+        name: dist-${{ matrix.os }}
+
+  build_wheels_extra:
+    name: Build extra wheels on ${{ matrix.os }} (${{ matrix.precision }}/${{ matrix.index }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
```

Should this be
```suggestion
        os: [ubuntu-latest, macos-latest, macos-15-intel, windows-latest]
```
like above or should the macOS steps below be removed?

---

I think we should add `macos-latest` but not the others for this "extra" case.